regexes/device/mobiles.yml

```
@@ -7835,7 +7863,7 @@ Orange:
     - regex: 'FunTab ([^/;]+) Build'
       model: 'FunTab $1'
       device: 'tablet'
-    - regex: 'Orange[ _-]([^/;]+) Build'
+    - regex: 'Orange[ _-]([^/;]+)( Build|\))'
```

When using `)` as "stop" character, it should be excluded from the match pattern, as it otherwise could match to much.
Would imho be better to use this
```suggestion
    - regex: 'Orange[ _-]([^/;)]+)( Build|\))'
```